site/src/docs/installation/from-source.md

```diff
@@ -18,7 +18,7 @@ cargo install cargo-nextest --locked
 
     A plain `cargo install cargo-nextest` without `--locked` is **not supported**. If you run into build issues, please try with `--locked` before reporting an issue.
 
-`cargo nextest` must be compiled and installed with **Rust {{ msrv }}** or later (see [_Stability policy_ for more), but it can build and run
+`cargo nextest` must be compiled and installed with **Rust {{ msrv }}** or later (see [_Stability policy_](../stability/index.md) for more), but it can build and run
```

Comment:
I think you can just have a closing bracket here, since it's defined at line 24 below.